src/lib.rs

```diff
@@ -27,6 +27,37 @@
 //!   setup_Rmainloop();
 //! }
 //! ```
+//!
+//! # Conditional compilation depending on R installation
+//!
+//! libR-sys crate provides these environmental variables that you can use in `build.rs`:
```

Comment:
I think this should be written as "The crate libR-sys provides ..."

Reply:
How about "The libR-sys crate provides ..."? I feel "The crate <crate name>" form seems uncommon. For example:

* "The crate serde provides": https://www.google.com/search?q="The+crate+serde+provides" (0 hit)
* "The serde crate provides": https://www.google.com/search?q="The+serde+crate+provides" (9 hits)

Reply:
Works for me.

Reply:
Thanks, let's use this.